ldms/src/decomp/static/decomp_static.c

```diff
@@ -1785,22 +1805,24 @@ static int decomp_static_decompose(ldmsd_strgp_t strgp, ldms_set_t set,
 			dup_row = row_cache_dup(cfg_row, mid_rbn, row);
 
 			/* Apply functional operators to columns */
+                        struct ldmsd_row_list_s tmp_row_list;
+                        int count;
+                        count = ldmsd_row_cache_make_list(&tmp_row_list,
```

Comment:
@morrone, the reason it was inside the col loop is to support operators that work with multiple columns, e.g. +, -, /. But we don't have any of those yet, so this change is fine.